### ISSUE TITLE
FORKTD ignoring second argument of run

### ISSUE THREAD

**Issue by artuuuro** [LLNL/qball#36]
Simulations with wave_wf = FORKTD give identical results even though second parameter of run is different.

for example:
set wf_dyn FORKTD
set TD_dt 0.001 atomictime
run 1 1
or
run 1 10
would give identical results

in both cases if I grep for etotal it will show only one etotal_int tag per run.

**Comment by correaa** [comment]
That's right, `nite_` doesn't seem to be used at all in https://github.com/LLNL/qball/blob/master/src/qball/EhrenSampleStepper.cc , so the value has no effect.
